src/code-generation/CodeGeneratorBase.ts

```diff
@@ -143,7 +156,7 @@ abstract class CodeGeneratorBase<TSettings extends CodeGeneratorBaseSettings> im
     if (!oneLineBranch) {
       this._branches.pop()
     }
-    this.writeLine(`${bracket} ${this.formatComment(comment, documentation)}`)
+    this.writeLine(`${this.formatComment(comment, documentation)}`)
```

Comment:
If Lua does not need closing brackets, you can override `getClosingBracket` and make it return an empty string. If you still want to show comments, part of this method has to be changed to accommodate this. If you can't develop a generic solution for this, override the `popBranchWithComment` in your class with the required behavior.

Reply:
Fixed